test/wasm-only.fromasm

```diff
@@ -266,6 +266,32 @@
    )
   )
  )
+ (func $f32-to-int64 (param $0 f32) (result i64)
```

Comment:
is this test changed because now we have a method that converts f32 to int, instead of just f64?

if so, why is there still usage of ensureDouble to force an f32 to an f64?

Reply:
Why is there still usage in general?
Because asm2wasm uses it elsewhere and I'm not sure what it does, and because the "js" trap mode calls a javascript import, which needs to be an `f64`

Reply:
Got it.